tests/test_utils.py

```diff
@@ -118,6 +121,27 @@ def test_regionprops():
         assert set(obj.properties.keys()) == set(properties)
 
 
+def test_extra_regionprops():
+    """Test adding a callable function for extra property calculation."""
+    img, centroids = create_test_image()
+
+    def extra_prop(_mask) -> float:
+        return np.sum(_mask)
+
+    extra_properties = (extra_prop,)
```

Comment:
Why not just something like this?
```suggestion
    extra_properties = (np.sum(_mask),)
```

Reply:
Happy to do it this way, but I was trying to emulate how an end user would actually use this. Probably doesn't matter in the tests

Reply:
The idea is that you can now provide functions that work on the image data and return properties that become part of the `PyTrackObject` properties - it's much cleaner to use and makes end-use much more powerful